pkgs/applications/video/kodi/addons/mediathekview/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, buildKodiAddon, fetchFromGitHub, addonUpdateScript, myconnpy }:
+
+buildKodiAddon rec {
+  pname = "mediathekview";
+  namespace = "plugin.video.mediathekview";
+  version = "1.0.9";
+
+  src = fetchFromGitHub {
+    owner = pname;
+    repo = namespace;
+    rev = "release-${version}";
+    hash = "sha256-XYyocXFTiYO7Ar0TtxjpCAy2Ywtnwb8BTxdKxwDWm4Y=";
+  };
+
+  propagatedBuildInputs = [
+    myconnpy
+  ];
+
+  passthru.updateScript = addonUpdateScript {
```

Comment:
this slipped by me... your other PR reminded me this isn't necessary

see https://github.com/NixOS/nixpkgs/pull/291065 as an example - `fetchFromGitHub` is enough to make the bot update the package

if you don't mind can you please remove this whenever you get a chance - it serves no purpose and can lead people down the wrong path

thank you and sorry for missing! 🙇‍♂️

Reply:
Good to know, thank you! I will change that.

Reply:
Btw, there are quite a lot of addons with this directive. Should we remove them too?

Reply:
that sounds like a mistake yeah

if a package was originally using the kodi repo for source and then switched to github i can see how that would get missed pretty easily